hack/ci/install-mce.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
```

Comment:
can we doc what it does?
```
Installs MCE from a custom catalog source, makes the OCP cluster a "local-cluster", and enables the hypershift operator.
```

Reply:
updated. I also added some details about prerequisites